README.md

```diff
@@ -11,19 +11,42 @@ These instructions will get you a copy of the project up and running on your loc
 
 ### Prerequisites
 
-Install Python 3 and pip
+To run the script (python or docker), you need to have an existing Google Agenda `credentials.json`.
+To obtain it, you'll have to create a google api account, add the Google Agenda API, then download the `credentials.json` file.
+See one of these links as an example documentation:
+
+- [Python Quickstart](https://developers.google.com/calendar/quickstart/python)
+- [Google API Authentication](https://flaviocopes.com/google-api-authentication/),
+
+Copy the `config-example.json` file to `config.json` and change it according to your needs (it should be documented enough for you to understand).
+Typically, you should have to change the used `dates` which contain the time slots for conferences.
+
+### Run with Docker
+
+Install Docker for your environment and run this command:
+
+```sh
+docker-compose up transform
```

Comment:
In fact, I would better have no docker-compose, as I find it quite overkill.

Reply:
Removed :)